src/test/ui/const-generics/concrete-const-impl-associated-non-method.rs

```diff
@@ -0,0 +1,18 @@
+// Test that an associated non-method within an impl block of a concrete const type i.e. A<2>, is
```

Comment:
I would merge this test with the `concrete-const-impl-method.rs` test, as they're essentially testing the same things.

Reply:
Done